packs/data/feats.db/crane-flutter.json

```diff
@@ -40,7 +40,7 @@
                 "key": "FlatModifier",
                 "predicate": [
                     "crane-flutter",
-                    "crane-stance"
+                    "self:effect:crane-stance"
```

Comment:
May not need this, as "crane-flutter" is false if not "self:effect:crane-stance" in the above RollOption.